### ISSUE TITLE
New rule: disallow _fixtures (without value) as parameters

### ISSUE THREAD

**Issue by m-burst** [m-burst/flake8-pytest-style#11]
Bad:
```py
def test_foo(_fixture):
    pass
```

Good:
```py
@pytest.mark.usefixtures('_fixture')
def test_foo():
    pass
```

Rationale: value of `_fixture` is always `None` per `PT004`/`PT005`
